client/src/test/runTest.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+
+import { runTests } from 'vscode-test';
+
+async function main() {
+	try {
+		// The folder containing the Extension Manifest package.json
+		// Passed to `--extensionDevelopmentPath`
+		const extensionDevelopmentPath = path.resolve(__dirname, '../../../');
+
+		// The path to test runner
+		// Passed to --extensionTestsPath
+		const extensionTestsPath = path.resolve(__dirname, './index');
+
+		// Download VS Code, unzip it and run the integration test

```

Comment:
Is it downloading, i.e., opens a network connection? It looks like it is grabbing the data from line 9.

Reply:
What do you mean saying grab data from line 9? I think these are all necessary code to setup the test.

Reply:
The comment says "Download" but the code seems to use the local files as defined in line 9. Is that correct? 

 I am asking to make sure that the code does not use the network to download anything.